foods/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+    parent_id = models.ForeignKey("self", related_name='categories', on_delete=models.SET_NULL, blank=True, null=True)
+
+
+class Food(models.Model):
+    category_id = models.ForeignKey(Category, related_name='foods', on_delete=models.CASCADE)
+    name = models.CharField(max_length=50)
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+    discount = models.IntegerField(default=0)
+    description = models.TextField()
+
+
+class Images(models.Model):
+    image_url = models.ImageField()
+    food_id = models.ForeignKey(Food, related_name='images', on_delete=models.CASCADE)
```

Comment:
You should create folder Category, Food, Images.. In the future, It is easy managment

Reply:
I'ld consider splitting it